### PR TITLE
Data API fixes

### LINKED ISSUE #26: [Data API] Verifiy endpoint account/balance
Check if endpoint logic is compliant with rosetta's specifications.
Check unit tests.

### PR DESCRIPTION
**Pull request by raynaudoe**
On this PR:

- [x] Improve account/balance: add `LockedBalance` and `SpendableBalance` info for account.
- [x] Improve network/status: add `sync_status` for network
- [x] Refactor rosetta error codes: use enums instead of strings.
- [x] Fix rosetta unit tests

Closes #26 #7

### CODE REVIEW

**Comment by jleni** [comment]
Is "Fix rosetta unit tests" still pending or this ready to be reviewed?

**Comment by raynaudoe** [comment]
I've fixed the network related ones, for reasons that I still don't know I cannot run any test that involves querying data to the db.
Trying to figure out a way of running all of this test locally.

File: src/api/rosetta-constants.ts [hunk 1]
```diff
@@ -11,6 +12,15 @@ export const RosettaConstants = {
   rosettaVersion: '1.4.6',
   symbol: 'STX',
   decimals: 6,
+  lockedBalance: 'LockedBalance',
+  spendableBalance: 'SpendableBalance',
+};
+
+export const ReferenceNodes: { [key: string]: any } = {
+  mainnet: {
```
Comment (rllola) [inline src/api/rosetta-constants.ts:20]: Do we have testnet info for this too ?

**Review by rllola — APPROVED** [review]

File: src/api/rosetta-constants.ts (hunk 1 above)
Comment (raynaudoe) [reply src/api/rosetta-constants.ts:20]: I've asked that [here](https://github.com/Zondax/stacks-blockchain-api/issues/25#issuecomment-791599981)
Still waiting for that info

**Pull request merged**